sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/HadoopFsRelation.scala

```diff
@@ -72,7 +72,8 @@ case class HadoopFsRelation(
     val compressionFactor = sqlContext.conf.fileCompressionFactor
     val defaultSize = (location.sizeInBytes * compressionFactor).toLong
     location match {
-      case cfi: CatalogFileIndex if sparkSession.sessionState.conf.fallBackToHdfsForStatsEnabled =>
+      case cfi: CatalogFileIndex if sparkSession.sessionState.conf.fallBackToHdfsForStatsEnabled &&
+        defaultSize == sqlContext.conf.defaultSizeInBytes =>
```

Comment:
indentation?

Reply:
@dongjoon-hyun I think, locally it is passing the scalastyle checks.

Reply:
Ya. I know. Officially, we need two more spaces. This is not covered by `dev/scalastyle`.

Reply:
Thanks. Done.